packages/checkout/core/src/components/malga-payments-drip/partials/malga-payments-drip-content/malga-payments-drip-content.service.ts

```diff
@@ -0,0 +1,47 @@
+import axios from 'axios'
+import {
+  formatISODate,
+  formatCurrency,
+  parseAmountDecimalToInteger,
+} from '@malga-checkout/utils'
+
+import { MalgaPaymentsDripContentConstructor } from './malga-payments-drip-content.types'
+
+export class MalgaPaymentsDripContentService {
+  readonly amount: number
+  readonly date: Date
+  readonly dripURL: string
+
+  constructor({ amount, debug, sandbox }: MalgaPaymentsDripContentConstructor) {
+    this.amount = amount / 100
+    this.date = formatISODate(new Date())
+    this.dripURL =
+      sandbox || debug
+        ? 'https://sbx-drip-be.usedrip.com.br/api/v1/instalments_simulator'
```

Comment:
Maybe we should move those constants to another place

Reply:
I like to maintain it here. I think it's better to keep these constants in this place, shared within the same context. Does it make sense to you?